File: problem/Count_Elements_Max_Frequency/maxFrequencyElements.c
int maxFrequencyElements(int *nums, int numsSize)
{
    int freq[101] = {0};
    int maxf = 0, res = 0;
    for (int i = 0; i < numsSize; ++i)
    {
        int n = nums[i];
        int f = ++freq[n];
        if (f > maxf)
        {
            maxf = f;
            res = f;
        }
        else if (f == maxf)
        {
            res += f;
        }
    }
    return res;
}

File: problem/Count_Elements_Max_Frequency/maxFrequencyElements.py
class Solution:
    def maxFrequencyElements(self, nums):
        freq = [0] * 101
        maxf = 0
        res = 0
        for n in nums:
            freq[n] += 1
            f = freq[n]
            if f > maxf:
                maxf = f
                res = f
            elif f == maxf:
                res += f
        return res
